demystifying_command_line/demystifying_command_line.md

```diff
@@ -0,0 +1,251 @@
+
+**Please don't let gatekeepers change your understanding of your own competence!** 
+
+</div>
+
+## Command Line Interface Common Questions
+
+If you've never used a command line, you might have a few common questions, like:
+
+* How do I get to the command line?  What program do I open in my computer to be able to type in commands?
+* What do I type into the command line to do things?  How will I know what to type?
+* What will happen if I get things wrong?
+
+Where is the Command Line Interface?
+----
+
+Most people use a Mac or a Windows operating system on their computer.  To get to the command line, you'll use **Terminal** in macOS or **Command Prompt** or **PowerShell** or **Git Bash** in Windows.  Using the command line can be a little different based on the computer system you use and which program you use. Terminal is a Linux-like environment that uses the bash language, while Command Prompt on Windows is limited to MS-DOS commands without scripting abilities.  PowerShell is an automation engine designed for Microsoft products that you can also run commands from, but we generally don't recommend it for working with Git.  And Git Bash is a Windows add-on that allows Windows users to have the same powerful and industry-standard Linux-like interface that Mac users get by default.
+
+What Do I Type?
+---
+
+When it comes to knowing what to do in a computer, the command line interface can feel daunting.  There's no help menu to click on, just a blank space that you're supposed to type into.  But what do you type?  How do you know what to do?  
+
+There are lots of places to learn commands.  One skill that will help is learning to effectively search the internet for a good description of the thing you're trying to do.  Chances are that you'll find someone who has already had the same challenge as you and learned how to solve it.  For example, if you're not sure how to list the files in a directory, you might search using terms like "command line what files are in my directory" or "CLI show files Windows" or something similar.  It takes practice to learn the vocabulary to create good search terms, but keep in mind that even pros who use the command line many times a day will often search for things rather than rely on their memory!  
+
+![A four-panel cartoon. People standing over a bomb with an attached keyboard yell "Rob! You use Unix! Come quick!".  The bomb display reads "To disarm the bomb, simply enter a valid `tar` command on your first try.  No googling.  You have ten seconds."  The group looks on in silence, until one person says, "Rob?".  Rob replies, "I'm so sorry."](https://imgs.xkcd.com/comics/tar.png "[Tar](https://xkcd.com/1168/) comic by xkcd, [CC BY-NC 2.5](https://xkcd.com/license.html).")
```

Comment:
```suggestion
![A four-panel cartoon. People standing over a bomb with an attached keyboard yell "Rob! You use Unix! Come quick!".  The bomb display reads "To disarm the bomb, simply enter a valid `tar` command on your first try.  No googling.  You have ten seconds."  The group looks on in silence, until one person says, "Rob?".  Rob replies, "I'm so sorry."](https://imgs.xkcd.com/comics/tar.png "[Tar](https://xkcd.com/1168/) comic by xkcd, [CC BY-NC 2.5](https://xkcd.com/license.html).")
```

Consider providing some (very very short) exegesis for this comic in the text. I think this can be a great way to point out that even experienced users of CLIs can't always write a successful command without any references, but without explanation (what is tar? what is unix?) it might fall a little flat.  

This could also fit well on the Mythology page :)

Reply:
I added a bit of additional context here!